Adding door enable anomaly

richardjedgar opened the pull request:
Adding the door_b_enb (spurious door side B enable) anomaly page.

Commit:
Adding door enable anomaly

jzuhone committed:
Merge branch 'master' into door_b_enb

Commit:
adding door_b_enb (door B enable) anomaly page

Commit:
Merge branch 'door_b_enb' of github.com:acisops/acis_anomaly_pages into door_b_enb

Merging into acis_anomaly_pages with door_b_enb branch

jzuhone commented on source/door_b_enb.rst:
```diff
@@ -0,0 +1,174 @@
+.. _dpaa-shutdown:
```
This link info should be changed :)

richardjedgar replied:
fixed (I think)

jzuhone commented on source/door_b_enb.rst:
```diff
@@ -0,0 +1,174 @@
+  to the thermistors being powered up when the mechanism is enabled. Side A: 1MAHOAT and 1MAHCAT.
+* 1DACTAT (the Door Angle Potentiometer) should be unchanged at 70 +/- 5 degrees,
+  indicating the door has not moved.
+* ACIS should still be recieving photons if a science run is in progress.
```
We might want to mention here that for really faint sources the count rate will not be that much above what we expect for the instrumental background if the door was closed (correct?)

richardjedgar replied:
Good point.

jzuhone commented on source/door_b_enb.rst:
```diff
@@ -0,0 +1,174 @@
+  is unpowered and reads CLOS (closed). Note this item does not appear on the ACIS
+  Real Time telemetry pages. The OC has access to this item on GRETA pages. Side A: 1MECLACL.
+
+All other hardware telemetry should be nominal. The current values for these (except 1MECLBCL)
```
I would add "and 1MECLACL" also here

jzuhone commented on source/door_b_enb.rst:
```diff
@@ -0,0 +1,174 @@
+FOT Procedures
+++++++++++++++
+
+* SAP 61032 (Open ACIS door) and
```
Do we not have links for these?

richardjedgar replied:
Apparently there are no links for these really ancient procedures. I imagine they're "retired".

jzuhone commented on source/door_b_enb.rst:
```diff
@@ -0,0 +1,174 @@
+valves were to close at this point in the mission, there would be no immediate
+consequences.
+
+See other anomaly pages for responses to spurious box turn-off commands (DPA-A, DPA-B, DEA-A).
```
It'd be nice to make the things in parentheses links here, but it's no big deal either way.

richardjedgar replied:
I'm sure there's a clever way to do that, beyond the brute-force one...?

Commit:
Addressing JAZ's comments

Commit:
Adding explicit @head to the sot_red_alert address

Commit:
Adding explicit @head to sot_yellow_alert addresses

Commit:
Fixed words about disabling an operating system

Commit:
Edits per Paul's comments this date

jzuhone commented on source/door_b_enb.rst:
```diff
@@ -73,8 +73,6 @@ Our real-time web pages will alert us and the Lead System Engineer will call us.
 ...possibly useful things left over from the DEA-A poweroff page I cribbed from... TBR
 
 * Identify whether or not additional comm time is needed and if so ask the OC/LSE to request it.
-* Send an email to ``sot_red_alert@head`` and call a telecon with the FOT, SOT, and FDs to brief
```
@richardjedgar was there a reason these lines got yanked? I suspect we talked about it but I don't remember.

jzuhone replied:
@richardjedgar ignore this comment, I see what happened now.

jzuhone commented on source/door_b_enb.rst:
```diff
@@ -49,38 +58,39 @@ What is the response?
+* Contact the GOT Duty Officer to inform that we need the dump data as soon as possible and to
+  email or call us when the dump file is available.
+* Process the dump data and make sure that there is nothing anomalous in the data *BEFORE*
+  the shutdown. We want to know if a new occurrence looks just like the previous occurrences.
```
This should probably say "before the anomaly", instead of "shutdown". I'm guessing that this was copied over from one of the shutdown pages.

jzuhone commented on source/door_b_enb.rst:
```diff
@@ -92,10 +102,13 @@ Impacts
 Relevant Procedures
 -------------------
-
+Note these are included for reference only. DO NOT EXECUTE these SOPs without consultaiton with
```
Misspelled "consultation"

Commit:
Fixed typos per JAZ's comments

jzuhone commented:
@richardjedgar I see some minor formatting issues that I will fix by hand from Github itself once everyone is satisfied.

jzuhone committed:
Minor formatting fix

cegrant commented:
Two very minor comments.  The "Impacts" section is weirdly formatted, at least in the rendered version I'm looking at.  Misspelled "inadvertently" in the final section.  Otherwise, looks ok to me.

jzuhone commented:
@cegrant I fixed the weird formatting earlier today, but it didn't make it into the rendered version.

richardjedgar commented:
Ah. I’ll recompile in the morning and re-post. 


> On Apr 24, 2018, at 8:07 PM, John ZuHone <notifications@github.com> wrote:
> 
> @cegrant <https://github.com/cegrant> I fixed the weird formatting earlier today, but it didn't make it into the rendered version.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/acisops/acis_anomaly_pages/pull/18#issuecomment-384118584>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AJWl_9H_U9MK6ezlDEKSlMOX9dNq2kI1ks5tr745gaJpZM4TKljQ>.
>

Commit:
Fixed spelling

richardjedgar commented:
I've fixed the spelling problem and recompiled, so that John's formatting
edit is now also visible. I copied again to the tmp disk area for your
convenience. There should be a new commit on the git page.

Richard Edgar, Chandra/ACIS operations
Smithsonian Astrophysical Observatory
o: 617-495-7249 c: 617-866-8615

On Tue, Apr 24, 2018 at 8:09 PM, Richard Edgar <redgar@cfa.harvard.edu>
wrote:

> Ah. I’ll recompile in the morning and re-post.
>
>
> On Apr 24, 2018, at 8:07 PM, John ZuHone <notifications@github.com> wrote:
>
> @cegrant <https://github.com/cegrant> I fixed the weird formatting
> earlier today, but it didn't make it into the rendered version.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/acisops/acis_anomaly_pages/pull/18#issuecomment-384118584>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJWl_9H_U9MK6ezlDEKSlMOX9dNq2kI1ks5tr745gaJpZM4TKljQ>
> .
>
>
>

jzuhone committed:
Merge branch 'master' into door_b_enb

Commit:
Addressed team comments as of May 3, 2018

Commit:
Add verification of door mechanism disabled after sending command.

richardjedgar commented:
If someone would like to merge this, I'll go ahead and install it. @Gregg140 @jzuhone

richardjedgar commented:
Deployed. Thanks, everybody!
